### ISSUE TITLE
Compile error does not show full stacktrace.

### ISSUE THREAD

**Issue by kesha0890** [Sketchware-Pro/Sketchware-Pro#240]
Hello, I am trying to compile another's project, but I got the next error: `No such file or directory`. It doesn't say me anything about error and it is quite hard to report even here or fix somehow.

I think skpro should show full stacktrace (at least from where it comes).

**Comment by kesha0890** [comment]
![Screenshot_20210909_213509_com sketchware remod](https://user-images.githubusercontent.com/60298573/132750725-b37da471-276b-4c3f-a877-8fb7689dc9fd.jpg)

**Comment by kesha0890** [comment]
Файл можно получить по ссылке:
X-Unity (Telegram).swb
https://disk.yandex.ru/d/DJnNLkj5LmKNfQ

**Comment by JavkhlanK** [comment]
yeah that might be good, but as fallback, the file /Internal storage/.sketchware/debug.txt contains more info, including the stack trace.

**Comment by JavkhlanK** [comment]
It currently does for unexpected exceptions, so those which aren't compile errors from e.g. aapt2, ecj, etc.
